README.md

```diff
@@ -9,7 +9,7 @@
 [![Slack](https://slack.rook.io/badge.svg)](https://slack.rook.io/badge.svg)
 [![Twitter Follow](https://img.shields.io/twitter/follow/rook_io.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=rook_io&user_id=788180534543339520)
 
-## What is Rook?
+# What is Rook
```

Comment:
The question mark should remain in the headin

Reply:
@galexrt brought it back.